fission-server/migrations/2023-05-16-122237_create_accounts/up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE accounts (
+  id SERIAL PRIMARY KEY,
+  did TEXT NOT NULL,
+  username TEXT NOT NULL,
+  verified BOOLEAN NOT NULL DEFAULT false,
```

Comment:
We can change all this later, so I'm not worried, but the current approach doesn't have a "verified" flag – we require verification to create an account.

Reply:
Ah right, will remove 👍